spec.md

```diff
@@ -1367,6 +1468,122 @@ Errors](#service-broker-errors).
 }
 ```
 
+## Fetching a Service Instance
+
+If `"instance_retrievable" :true` is declared for a service in the
+[Catalog](#catalog-management) endpoint, brokers MUST support this endpoint for
+all plans of the service and this endpoint MUST be available immediately after
+the
+[Polling Last Operation for Service Instances](#polling-last-operation-for-service-instances)
+endpoint returns `"state": "succeeded"` for a [Provisioning](#provisioning)
+operation.
+
+### Request
+
+##### Route
+`GET /v2/service_instances/:instance_id`
+
+`:instance_id` is the identifier of a previously provisioned instance.
+
+##### cURL
+```
+$ curl 'http://username:password@broker-url/v2/service_instances/:instance_id' -X GET -H "X-Broker-API-Version: 2.11"
+```
+
+### Response
+
+| Status Code | Description |
+| --- | --- |
+| 200 OK | The expected response body is below. |
+| 404 Not Found | MUST be returned if the service instance does not exist or if a provisioning operation is still in progress. The expected response body is `{}`. |
```

Comment:
What should happen if the instance **update** request is in progress (i.e. there is an existing resource, but the update processing is not finished yet)?

Reply:
Seems to me like this needs to be outlined here as a `422 - Unprocessable Entity` situation.

Reply:
Done

Reply:
I wonder if 409 is more appropriate:
409 Conflict
    Indicates that the request could not be processed because of conflict in the request, such as an edit conflict between multiple simultaneous updates.